### PR TITLE
Add MultiMarkdown Composer Pro Beta 3.0b46

### PR DESCRIPTION
**Pull request by ickc**
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-versions/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-versions/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Not finding a download link for this beta on the website. Where is it?

**Comment by ickc** [comment]
should have mention it: http://support.multimarkdown.com/kb/future-plans/where-can-i-download-the-beta-for-v3

File: Casks/multimarkdown-composer-pro.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'multimarkdown-composer-pro' do
+  version '3.0b46'
+  sha256 '9eee8954e42d705b8a3433ef07ff52394b7e910c02ad0df9b4fa1dbfadee4b96'
+
+  # amazonaws.com was verified as official when first introduced to the cask
+  url "http://files.fletcherpenney.net.s3.amazonaws.com/MultiMarkdown%20Composer%20Pro%20%28Beta%29-#{version}.zip"
+  name 'MultiMarkdown Composer Pro (Beta)'
```
Comment (vitorgalvao) [inline Casks/multimarkdown-composer-pro.rb]: Remove ` (Beta)`.

File: Casks/multimarkdown-composer-pro.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'multimarkdown-composer-pro' do
+  version '3.0b46'
+  sha256 '9eee8954e42d705b8a3433ef07ff52394b7e910c02ad0df9b4fa1dbfadee4b96'
+
+  # amazonaws.com was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/multimarkdown-composer-pro.rb]: This is too unspecific. Needs to be `files.fletcherpenney.net.s3.amazonaws.com`.

File: Casks/multimarkdown-composer-pro.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  url "http://files.fletcherpenney.net.s3.amazonaws.com/MultiMarkdown%20Composer%20Pro%20%28Beta%29-#{version}.zip"
+  name 'MultiMarkdown Composer Pro (Beta)'
+  homepage 'http://support.multimarkdown.com/kb/future-plans/where-can-i-download-the-beta-for-v3'
```
Comment (vitorgalvao) [inline Casks/multimarkdown-composer-pro.rb]: Thumbs up to this change. Makes it clearer.

**Comment by vitorgalvao** [comment]
It’s a bit weird to have the beta here and no stable version on the main repo. Mind adding version 2 in [caskroom/cask](https://github.com/caskroom/homebrew-cask)?

File: Casks/multimarkdown-composer-pro.rb (hunk 2 above)
Comment (ickc) [inline Casks/multimarkdown-composer-pro.rb]: done

File: Casks/multimarkdown-composer-pro.rb (hunk 1 above)
Comment (ickc) [inline Casks/multimarkdown-composer-pro.rb]: done

**Comment by ickc** [comment]
I'll add the non beta and non pro version probably in 2 days. Thanks.

**Comment by vitorgalvao** [comment]
Thank you. The non-beta pro stable version should be added here. The non-beta stable non-pro version should be in the main repo.

**Comment by ickc** [comment]
Added in caskroom/homebrew-cask#24864 and #2660.

v2.9 only has non-pro version. So in homebrew cask there's MMDC v2.9 and in homebrew version there are MMDC v3.0b46 and MMDC Pro v3.0b46.

**Pull request merged**